### DISCUSSION TITLE
Deploying Nextjs app failed on versel

### DISCUSSION THREAD

**Discussion by EECvision** [vercel/next.js#16915] [Help]
Hi, I am trying to deploy my first Nextjs blog app on versel and I got the error message, Deployment failed. Please, what should I do to resolve this? I have also attached the logs. Thanks

19:12:38 | Cloning github.com/EECvision/nextjs-blog (Branch: master, Commit: e7c46e2)
-- | --
19:12:39 | Cloning completed in 1164ms
19:12:39 | Analyzing source code...
19:12:40 | Installing build runtime...
19:12:40 | Build runtime installed: 444.825ms
19:12:41 | Looking up build cache...
19:12:41 | Build cache not found
19:12:41 | Installing dependencies...
19:12:52 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
19:12:52 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:12:52 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
19:12:52 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:12:52 | added 857 packages from 350 contributors in 10.256s
19:12:52 | 80 packages are looking for funding
19:12:52 | run `npm fund` for details
19:12:52 | Running "npm run build"
19:12:53 | > learn-starter@0.1.0 build /vercel/5b20c43c
19:12:53 | > next build
19:12:53 | Browserslist: caniuse-lite is outdated. Please run next command `npm update`
19:12:53 | Warning: No build cache found. Please configure build caching for faster rebuilds. Read more: https://err.sh/next.js/no-cache
19:12:53 | Creating an optimized production build...
19:12:53 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
19:12:53 | This information is used to shape Next.js' roadmap and prioritize features.
19:12:53 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
19:12:53 | https://nextjs.org/telemetry
19:13:00 | Failed to compile.
19:13:00 | ./pages/posts/[id].js
19:13:00 | Module not found: Can't resolve '../../components/layout' in '/vercel/5b20c43c/pages/posts'
19:13:00 | > Build error occurred
19:13:00 | Error: > Build failed because of webpack errors
19:13:00 | at build (/vercel/5b20c43c/node_modules/next/dist/build/index.js:13:900)
19:13:00 | npm ERR! code ELIFECYCLE
19:13:00 | npm ERR! errno 1
19:13:00 | npm ERR! learn-starter@0.1.0 build: `next build`
19:13:00 | npm ERR! Exit status 1
19:13:00 | npm ERR!
19:13:00 | npm ERR! Failed at the learn-starter@0.1.0 build script.
19:13:00 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
19:13:00 | npm ERR! A complete log of this run can be found in:
19:13:00 | npm ERR!     /vercel/.npm/_logs/2020-09-07T18_13_00_306Z-debug.log
19:13:00 | Error: Command "npm run build" exited with 1
19:13:02 | Done with "package.json"

**Comment by c0b41** [comment]
looks like layout import wrong, clear local cache and try.
